layouts/default.vue

```diff
@@ -91,28 +92,43 @@ export default {
       activeLocationIds: state => state.map.activeLocationIds,
     }),
     ...mapGetters('map', [
-      'activeSpatialData',
+      'activeRasterData',
       'activeVectorData',
       'activeDatasetsLocations',
       'datasetsInActiveTheme',
       'activeTimestamp',
       'activeDatasets',
       'getActiveTheme',
     ]),
-    spatialLayer() {
-      const spatialLayer = getSpatialLayer()
-      spatialLayer.source.tiles = this.activeSpatialData
-      return spatialLayer
+    rasterLayer() {
+      const rasterLayer = getRasterLayer()
+      rasterLayer.source.tiles = this.activeRasterData
+      return rasterLayer
     },
     vectorLayers() {
       const vectorLayers = this.activeVectorData
       const defaultVectorLayer = getVectorLayer()
-      vectorLayers.forEach(layer => {
+
+      const merged = _(vectorLayers)
```

Comment:
comments? source?

Reply:
comments in https://github.com/openearth/dgds-front-end/pull/30